docs/Contributing.asciidoc

```diff
@@ -143,7 +142,7 @@ mechanism, so it must be available on the system.
 The openQA package is built on top of Mojolicious, an excellent Perl framework
 for web development that will be extremely familiar to developers coming from
 other modern web frameworks like Sinatra and that have nice and comprehensive
-documentation available at its http://mojolicio.us[home page].
+documentation available at its https://mojolicious.org/[home page].
```

Comment:
```suggestion
documentation available at its https://mojolicious.org/[mojolicious home page].
```

Reply:
If we are talking about Mojolicious in this paragraph, then `its Mojolicious home page` sounds weird.